crates/kumod/src/queue.rs

```diff
@@ -142,6 +142,29 @@ impl ScheduledMetrics {
         self.by_tenant.as_ref().map(|m| m.sub(amount));
         self.by_tenant_campaign.as_ref().map(|m| m.sub(amount));
     }
+
+    pub fn remove_metrics_for_queue(name: &str) {
+        let components = QueueNameComponents::parse(name);
```

Comment:
Can remove this bit because it was only used to facilitate the case that I think we should not be doing

```suggestion
```